kubernetes/config/kube_config_test.py

```diff
@@ -120,6 +126,13 @@ def test_data_given_file(self):
         t = FileOrData(obj=obj, file_key_name=TEST_FILE_KEY)
         self.assertEqual(TEST_DATA_BASE64, t.as_data())
 
+    def test_data_given_file_no_base64(self):
+        obj = {
+            TEST_FILE_KEY: self._create_temp_file(content=TEST_DATA)}
+        t = FileOrData(obj=obj, file_key_name=TEST_FILE_KEY,
+                       base64_file_content=False)
+        self.assertEqual(TEST_DATA, t.as_data())
+
```

Comment:
So, I am not able to find test case for base64_file_content=True. I might have missed something here as I am new to the codebase. One more thing, what's the difference between this test-case and the one above, I think, since base64_file_content=False by default, the -ve scenario may be same as the old one. Please correct me, if I am wrong.